src/views/AccessControl.tsx

```diff
@@ -252,11 +260,11 @@ export const AccessControl = () => {
     }
 
     const renderPopoverGroups = (label: string, groups:Group[] | string[] | null) => {
-        const content = groups?.map(g => {
+        const content = groups?.map((g, i) => {
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
    const content = groups?.map((g, i) => {
```